src/components/modals/overwrite-map-modal.tsx

```diff
@@ -25,7 +25,8 @@ import {UploadAnimation} from './status-panel';
 import ImageModalContainer from './image-modal-container';
```

Comment:
unrelated...but saw this duplicated import on line 25 & 27. it can be merged into one line